cmd/sku/put.go

```diff
@@ -80,5 +87,7 @@ func newPutCommand(sl service.CommandServicer) (*cobra.Command, error) {
 		return cmd, err
 	}
 
+	cmd.Flags().BoolVarP(&oArgs.Force, "force", "", false, "Overwrite existing SKU if a SKU with the same ID already exists")
```

Comment:
note I didn't use `-f` because it's already in use by --sku-file and I didn't want to cause unexpected behavior by changing the meaning of `-f ` to force overwrite.